text/0000-docs-team.md

```diff
@@ -0,0 +1,107 @@
+
+# Detailed design
+[design]: #detailed-design
+
+The Rust documentation team will be responsible for all of the things listed above. Specifically, they will pertain
+to these areas of the Rust project:
+
+* The standard library documentation
+* The book and other long-form docs
+* Cargo's documentation
+* The Error Index
+
+Furthermore, the documentation team will be available to help with ecosystem documentation, in a few ways. Firstly,
+in an advisory capacity: helping people who want better documentation for their crates to understand how to accomplish
+that goal. Furthermore, monitoring the overall ecosystem documentation, and identifying places where we could contribute
+and make a large impact for all Rustaceans. If the Rust project itself has wonderful docs, but the ecosystem has terrible
+docs, then people will still be frustrated with Rust's documentation situation, especially given our anti-batteries-included
+attitude. To be clear, this does not mean _owning_ the ecosystem docs, but rather working to contribute in more ways
+than just the Rust project itself.
+
+We will coordinate in the `#rust-docs` IRC room, and have regular meetings, as the team sees fit. Regular meetings will be
+important to coordinate broader goals; and participation will be important for team members. We hold meetings weekly.
+
+## Membership
+
+* @steveklabnik, team lead
+* @GuillaumeGomez
+* @jonathandturner
+* @peschkaj
+
+It's important to have a path towards attaining team membership; there are some other people who have already been doing
+docs work that aren't on this list. These guidelines are not hard and fast, however, anyone wanting to eventually be a
+member of the team should pursue these goals:
+
+* Contributing documentation patches to Rust itself
+* Attending doc team meetings, which are open to all
+* generally being available on IRC to collaborate with others
+
+I am not quantifying this exactly because it's not about reaching some specific number; adding someone to the team should
+make sense if someone is doing all of these things.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This is Yet Another Team. Do we have too many teams? I don't think so, but someone might.
+
+# Alternatives
+[alternatives]: #alternatives
+
+The main alternative is not having a team. This is the status quo, so the situation is well-understood.
+
+It's possible that docs come under the purvew of "tools", and so maybe the docs team would be an expansion
+of the tools team, rather than its own new team. Or some other subteam.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+Should we quantify the number of commits before you can get on the team? Or the amount of time?
+
+Rustdoc is a tool that's owned by the tools team, but the doc team makes
+significant contributions to it. Is it part of the tools team or the docs team?
+Maybe a backend-frontend split? Or maybe it just stays with tools?
```

Comment:
I'd prefer to leave it with tools - it is a tool that generates documentation, not a piece of documentation. Given that we are not super-strict about who r+s what, I don't think it will make a big difference in practice - certainly not enough to warrant a frontend/backend split.